tools/tracevis/trace.html

```diff
@@ -72,6 +72,9 @@ <h4 style="text-align: center;">ParSeq JSON Trace Viewer vX.X.X</h4>
           <label class="checkbox inline">
             <input id="checkboxExcludeParent" type="checkbox" checked="checked" onclick="refreshView();">Parent tasks
           </label>
+          <label class="checkbox inline">
+            <input id="checkboxExcludeNonCritical" type="checkbox" checked="checked" onclick="refreshView();">Tasks outside of the critical path
```

Comment:
Would be better to have "Critical Path" that is unchecked by default - IMO it is shorter and more intuitive